docs/welcome/welcome-to-elrond.md

```diff
@@ -9,7 +9,7 @@ title: Welcome to Elrond
 - **When:** Mainnet live, project started in 2018
 - **How:** Scales via Adaptive State Sharding, consensus via Secure Proof of Stake
 - **Highlights:**
-  - Performance: 15.000 current TPS, 6s block time, $0.001 tx cost, can scale beyond 100k (263,000 TPS in testnet)
+  - Performance: 15,000 current TPS, 6s block time, $0.001 tx cost, up to 100,000 (263,000 TPS in testnet)
```

Comment:
I think that "beyond" was the right term there, "up to" sounds like a limit which is not true

Reply:
I changed that to try the comment-commit comments, but apparently didn't work. Let me try again.